docs/forgeops_apply_apps.md

```diff
@@ -0,0 +1,60 @@
+## forgeops apply apps
+
+Deploy the ForgeRock apps (AM, IDM, UI)
+
+### Synopsis
+
+
+    Deploy the ForgeRock apps (AM, IDM, UI):
+    * Apply the ForgeRock apps
+    * use --tag to specify a different version to deploy
+
+```
+forgeops apply apps [flags]
+```
+
+### Examples
+
+```
+
+      # Deploy the ForgeRock apps listed in the "latest" release of Forgeops.
+      forgeops apply apps
+
+      # Deploy the ForgeRock apps listed in a specific release of Forgeops.
+      forgeops apply apps --tag 2020.10.28-AlSugoDiNoci
+```
+
+### Options
+
+```
+  -h, --help   help for apps
+```
+
+### Options inherited from parent commands
+
+```
+      --as string                      Username to impersonate for the operation
+      --as-group stringArray           Group to impersonate for the operation, this flag can be repeated to specify multiple groups.
```

Comment:
May be a non-issue, but in any case: If multiple groups are assigned, will this use the "union" (most permissible) or "intersection" (least permissible) permission?

Reply:
these flags are inherited from a k8s library.